pkg/paravirt/paravirt_protected_entity_type_manager.go

```diff
@@ -381,47 +475,11 @@ func (this *ParaVirtProtectedEntityTypeManager) CreateFromMetadata(ctx context.C
 	clone.Status.ResourceHandle = svcClone.Status.ResourceHandle.DeepCopy()
 	_, err = this.gcBackupDriverClient.CloneFromSnapshots(cloneFromSnapshotNamespace).UpdateStatus(context.TODO(), clone, metav1.UpdateOptions{})
```

Comment:
not part of your code in this commit, but the style of error checking mentioned can also be applied here.